tests/integration/colin_tests.py

```diff
@@ -0,0 +1,12 @@
+import colin
+
+result = colin.run("colin-test")
```

Comment:
out of scope of this PR, but would be awesome to have a dedicated set of images for testing; will create a new issue